helm/cluster-vsphere/values.yaml

```diff
@@ -45,6 +28,30 @@ controlPlane:
   resourceRatio: 8
 
 global:
+  connectivity:
+    baseDomain: k8s.test
```

Comment:
This is a weird default to have isn't it?

Reply:
to be honest i'm not too sure where that comes from - i'll drop the default as it doesn't really make sense